src/main/webapp/browse-recipes.js

```diff
@@ -0,0 +1,28 @@
+function getRecipes(algorithm) {
```

Comment:
I thought there was only supposed to be one js file in the project.

Reply:
Oops, that's right. I'll combine them all into one file before I merge this feature branch into master.

Reply:
I'll combine them at that step because if I change them now, I'll have to rebase every sub-feature branch to reflect the change, which will almost certainly give me nightmares.